### PR TITLE
Add graphs to airflow-business-workflow-hightouch

### PR DESCRIPTION
**Pull request by llexan**
Adds images of the "Graph" views from these two Registry pages:
https://registry.astronomer.io/dags/modern-elt-salesforce-to-snowflake
https://registry.astronomer.io/dags/modern-elt-salesforce-to-snowflake-hightouch

To this blog post:
https://www.astronomer.io/blog/airflow-business-workflow-hightouch

### CODE REVIEW

**Review by jwitz — APPROVED** [review]
LGTM save for one bit @llexan : Very cool that you turned the images into hyperlinks (didn't know you could do that!) but I'm not sure it makes sense to do that when we already link out to the same page in the leading paragraphs for the images. As a user, that might be a little disorienting, and I usually expect an image to pop out in its own screen vs. open a new website.

**Comment by llexan** [comment]
Thanks, @jwitz! That's a good point, went ahead and removed the links from the images. 👍 

Also looks like we need a codeowner to approve 😅. @petedejoy @paolaperaza @sunkickr do you mind also approving if every thing looks good? Thank you all so much! 🙏

**Review by paolaperaza — APPROVED** [review]

**Pull request merged**
